### PR TITLE
Added TTL 74245

### PR DESCRIPTION
**Pull request by btb**
split from #1461

### CODE REVIEW

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74245.java [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+package com.cburch.logisim.std.ttl;
+
+import com.cburch.logisim.data.Value;
+import com.cburch.logisim.instance.InstancePainter;
+import com.cburch.logisim.instance.InstanceState;
+import com.cburch.logisim.util.GraphicsUtil;
+import java.awt.Graphics;
+
+/**
+ * TTL 74x240 octal buffers and line drivers with three-state outputs
```
Comment (maehne) [inline src/main/java/com/cburch/logisim/std/ttl/Ttl74245.java:19]: "74x240" -> "74x245"

**Review by maehne — CHANGES_REQUESTED** [review]
Looks mostly good to me!

As a side note: The TTL library lacks at the moment documentation. Help in this direction would be welcome, too.

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74245.java (hunk 1 above)
Comment (btb) [reply src/main/java/com/cburch/logisim/std/ttl/Ttl74245.java]: fixed

**Review by MarcinOrlowski — APPROVED** [review]

**Review by maehne — APPROVED** [review]

**Review by BFH-ktt1 — APPROVED** [review]
Please fix the conflict

**Review by BFH-ktt1 — APPROVED** [review]

**Pull request merged**